README.md

```diff
@@ -70,6 +70,46 @@ Almost everything can be automated through scripts. Process like deploy and dist
 
 Experiences and knowledge must be shared, it's the only way to grow. We love to share what we've learnt so far. We can organize talks or workshops designed for the client needs.
 
+## Design process of a digital product
+
+##### Branding
+
+- Research of visual referents and inspiration from different branding examples.
+- Design of several proposals based on the previous research and iteration of them.
+- Creation of a _____ based on the logo.
+- Definition of brand colors.
+- Selection of the brand font family.
+- Creation of different applications for the branding elements (social networks profiles, business cards, merchandising...).

```

Comment:
If this is a process and has an order I would use `1.   2.   3. ` instead of bullets.